packages/server-core/src/projects/scene/scene.class.ts

```diff
@@ -128,21 +128,26 @@ export class Scene implements ServiceMethods<any> {
       scenes[index].thumbnailUrl += `?${Date.now()}`
     }
 
-    return { data: scenes }
+    return scenes
```

Comment:
@hanzlamateen how should we be standardizing these in preparation for our hook refactor?

Reply:
@HexaField since this is a non-database based service. Mean it implements `ServiceInterface` therefore it can be left as it is. Because there is no `super.` or underscore methods in this case.

If required `ServiceInterface` based classes can be moved to hooks but that wouldnt make much sense.